src/dev-app/chips/chips-demo.html

```diff
@@ -172,17 +175,47 @@ <h4>Input is next sibling child of chip grid</h4>
       <mat-form-field>
         <mat-label>New Contributor...</mat-label>
         <mat-chip-grid #chipGrid2 [(ngModel)]="selectedPeople" required [disabled]="disableInputs">
-          <mat-chip-row *ngFor="let person of people" (removed)="remove(person)">
+          <mat-chip-row *ngFor="let person of people" [editable]="true" (removed)="remove(person)">
             {{person.name}}
             <button matChipRemove aria-label="Remove contributor">
               <mat-icon>cancel</mat-icon>
             </button>
           </mat-chip-row>
         </mat-chip-grid>
-        <input [matChipInputFor]="chipGrid2"
-               [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
-               [matChipInputAddOnBlur]="addOnBlur"
-               (matChipInputTokenEnd)="add($event)" />
+        <input
+          [matChipInputFor]="chipGrid2"
+          [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+          [matChipInputAddOnBlur]="addOnBlur"
+          (matChipInputTokenEnd)="add($event)"
+        />
+      </mat-form-field>
+
+      <h4>Input is last child of chip grid with Reactive Forms</h4>
```

Comment:
The approach with Reactive Forms is slightly different from the one without as you are working with form control. That's why I thought that it would be better to have both versions in example. If you think that it's not worth it I can take it out.